src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
+
+import duke.Storage;
+import duke.task.Task;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * The DeleteCommand class represents the delete command given by user.
+ */
+public class DeleteCommand extends Command {
+    private final int position;
```

Comment:
I like the usage of the term "position" as it indicates that you are referring to the position (starting from index 1) instead of the common "index" (starting from index 0) which most programmers like to use. Using the position term clarifies the need of when you need to subtract by 1.